File: Backend/WebServer/Python_Flask/backend/app/services/user_service.py
from app.models.user import User

# Dummy data store
_users = {
    1: User(1, "Alice"),
    2: User(2, "Bob")
}

def get_user_by_id(user_id):
    return _users.get(user_id)

def get_all_users():
    return list(_users.values())

File: Backend/WebServer/Golang_Hertz/backend/service/user_service.go
package service

import "backend/model"

var users = []model.User{
    {ID: 1, Username: "Alice"},
    {ID: 2, Username: "Bob"},
}

func GetAllUsers() []model.User {
    return users
}

func GetUserByID(id int) *model.User {
    for _, user := range users {
        if user.ID == id {
            return &user
        }
    }
    return nil
}
